docs/cis_self_assessment.md

```diff
@@ -1467,29 +1467,19 @@ Ensure that the `--cert-file` and `--key-file` arguments are set as appropriate
 etcd is a highly-available key value store used by Kubernetes deployments for persistent storage of all of its REST API objects. These objects are sensitive in nature and should be encrypted in transit.
 </details>
 
-**Result:** Pass
-
-**Remediation:**
-Follow the etcd service documentation and configure TLS encryption.
-Then, edit the etcd pod specification file `/etc/kubernetes/manifests/etcd.yaml`
-on the master node and set the below parameters.
-
-``` bash
---cert-file=</path/to/ca-file>
---key-file=</path/to/key-file>
-```
+**Result:** **Not Applicable**
```

Comment:
Let's mark this as Pass and update the audit and remediation to reflect that we use a config file and that is what should be checked, not command line arguments.